### ISSUE TITLE
namespaces declared reduntantly

### ISSUE THREAD

**Issue by sydb** [TEIC/Stylesheets#411]
Note: this is similar to, but not the same as, #118, I think.

In those cases where an example has a root element of `<TEI>` and descendant `<TEI>`s (i.e. `//teix:egXML/teix:TEI//teix:TEI`), those descendant `<TEI>` also (inappropriately) get decorated with an `@xmlns` namespace declaration in the HTML output. Note that this does **not** happen if the root element is `<teiCorpus>`.

(BTW, at the moment this _never_ happens in the released or dev branch _Guidelines_, only in the sydb-standOff branch.)

**Comment by sydb** [comment]
Update: now that `<TEI>` can nest in `<TEI>`, this _does_ occur in the dev and release branches. (And in the published _Guidelines_.)

**Comment by ebeshero** [comment]
@sydb @raffazizzi : Can we close this ticket? I am thinking this is no longer a problem.

**Comment by sydb** [comment]
No, this is still a problem, so I do not think this ticket should be PARKED. That said, the examples that show this are not _wrong_, they just show a redundant (and ugly) namespace declaration. (See, e.g., the 2nd or 3rd example in [16.1](https://www.tei-c.org/Vault/P5/4.9.0/doc/tei-p5-doc/en/html/CC.html#CCDEF), the only example in [16.3.2](https://www.tei-c.org/Vault/P5/4.9.0/doc/tei-p5-doc/en/html/CC.html#CCAS1), or the only example in [17.11](https://www.tei-c.org/Vault/P5/4.9.0/doc/tei-p5-doc/en/html/SA.html#SASOann)). So I think this can remain LOW PRIORITY.
